app/controllers/reactions_controller.rb

```diff
@@ -63,14 +63,16 @@ def create
         reactable_type: params[:reactable_type],
         category: category,
       )
-      if reaction.save
-        result = "create"
-        Moderator::SinkArticles.call(reaction.user_id) if vomit_reaction_on_user?(reaction)
-        Notification.send_reaction_notification(reaction, reaction.reactable.user)
-        Notification.send_reaction_notification(reaction, reaction.reactable.organization) if organization_article?(reaction)
-      else
-        return render json: { error: reaction.errors.full_messages.join(", "), status: 422 }, status: :unprocessable_entity
+
+      unless reaction.save
+        render json: { error: reaction.errors.full_messages.join(", "), status: 422 }, status: :unprocessable_entity
+        return
       end
+
+      result = "create"
+      Moderator::SinkArticles.call(reaction.user_id) if vomit_reaction_on_user?(reaction)
```

Comment:
This should be `reactable_id`:
```suggestion
      Moderator::SinkArticles.call(reaction.reactable_id) if vomit_reaction_on_user?(reaction)
```